internal/pkg/service/common/distribution/node.go

```diff
@@ -55,13 +53,13 @@ func NewNode(d dependencies, opts ...NodeOption) (*Node, error) {
 
 	// Create instance
 	n := &Node{
-		assigner: newAssigner(d.Process().UniqueID()),
-		clock:    d.Clock(),
-		logger:   d.Logger().AddPrefix("[distribution]"),
-		proc:     d.Process(),
-		schema:   d.Schema(),
-		client:   d.EtcdClient(),
-		config:   c,
+		assigner:    newAssigner(d.Process().UniqueID()),
+		groupPrefix: etcdop.NewPrefix(fmt.Sprintf("runtime/distribution/group/%s/nodes", group)),
```

Comment:
A tu je zmena v generovani etcd prefixu, uz sa nevola `buffer/store/schema`, ale generuja sa to priamo tu.